docs/topics/OAuth2.md

```diff
@@ -78,7 +78,13 @@ A URL can be generated that redirects authenticated users to the add-bot flow, b
 https://discordapp.com/api/oauth2/authorize?client_id=157730590492196864&scope=bot&permissions=0
```

Comment:
Let's specify these flows a bit more cleanly as well:

A URL can be generated that redirects authenticated users to the add-bot flow. It can be in the following format without a `redirect_uri`:
 
 ```https://discordapp.com/api/oauth2/authorize?client_id=157730590492196864&scope=bot&permissions=0```

Or with a `redirect_uri`:

```https://discordapp.com/api/oauth2/authorize?client_id=157730590492196864&scope=bot&permissions=0&redirect_uri=https%3A%2F%2Fnicememe.website```